File: QViz.NET/Periscope.QViz/JSON/Authentication.cs
namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for User Authentication
	/// </summary>
	public class Authentication
	{
		/// <summary>
		/// Unique Name of the User for Authentication
		/// </summary>
		public string username;
		
		/// <summary>
		/// Password of the User for Authentication
		/// </summary>
		public string password;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public Authentication()
		{
			this.username = "";
			this.password = "";
		}

		/// <summary>
		/// Create a new Authentication instance with provided User Name and Password
		/// </summary>
		/// <param name="username">Unique Name of the User for Authentication</param>
		/// <param name="password">Password of the User for Authentication</param>
		public Authentication(string username, string password)
		{
			this.username = username;
			this.password = password;
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/Authentication.java
package com.periscope.qviz.json;

/**
 * JSON Model for User Authentication
 */
public class Authentication {

	/**
	 * Unique Name of the User for Authentication
	 */
	public String username;

	/**
	 * Password of the User for Authentication
	 */
	public String password;

	/**
	 * Default Constructor
	 */
	public Authentication() {
		this.username = "";
		this.password = "";
	}

	/**
	 * Create a new Authentication instance with provided User Name and Password
	 *
	 * @param username Unique Name of the User for Authentication
	 * @param password Password of the User for Authentication
	 */
	public Authentication(String username, String password) {
		this.username = username;
		this.password = password;
	}

}
